pyvenv.el

```diff
@@ -189,7 +190,9 @@ This is usually the base name of `pyvenv-virtual-env'.")
                                 ;; No "=" means to unset
                                 "PYTHONHOME")
                                process-environment)
-          ))
+          )
+    ;; set eshell path to same as exec-path
+    (setq eshell-path-env (mapconcat 'identity exec-path ":")))
```

Comment:
You can put this into the other `setq` form as another variable

Reply:
Oh, and apparently we do not set this back to the original value on deactivation. Is that intended? (I do not use eshell and do not know the expected behavior)

Reply:
Ah, right; I didn't know off the top of my head if a value set in `setq` could be used in the same `setq` in a later line. The previous `exec-path` is restored on line 220 so later in `pyvenv-deactivate` I set `eshell-path-env` to that restored value.

Reply:
Should I put the `setq` in the previous statement?

Reply:
Two `setq` in sequence do not make sense, so merging the two into a single statement makes sense.

I don't know anything about eshell, so no idea if `eshell-path-env` ever should be different from `exec-path`, but if not, this works for me.

Reply:
Oh, d'oh, yeah, that's a good point about `eshell-path-env`. For me, it's always the same but I guess that's not always guaranteed. I'll rework this.